packages/@webex/internal-plugin-ai-assistant/src/ai-assistant.ts

```
@@ -0,0 +1,447 @@
+              message: concatenatedMessage,
+              requestId,
+              finished: true,
+              errorMessage: errorMessage || decryptError.message,
+              errorCode,
+            });
+          }
+        } else {
+          // For non-finished messages, concatenate and emit the accumulated message
+          try {
+            let decryptedMessage = '';
+            if (resultData && resultData.value) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)